### PR TITLE
[13.0][MIG] account_invoice_inter_company

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Standard migration of module account_invoice_inter_company to v13.

cc ~ @ForgeFlow

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Functional test in local, LGTM

File: account_invoice_inter_company/tests/inter_company_invoice.xml [hunk 1]
```diff
@@ -371,20 +391,38 @@
...
     </record>
-    <record id="customer_invoice_line_company_a" model="account.invoice.line">
-        <field name="invoice_id" ref="customer_invoice_company_a" />
-        <field name="product_id" ref="product_consultant_multi_company" />
-        <field name="name">Service Multi Company</field>
-        <field name="account_id" ref="a_sale_company_a" />
-        <field name="price_unit" eval="450.0" />
-        <field name="company_id" ref="company_a" />
-    </record>
+    <!--    <record id="customer_invoice_company_a" model="account.move">-->
```
Comment (AaronHForgeFlow) [inline account_invoice_inter_company/tests/inter_company_invoice.xml]: Better to remove this?

**Review by AaronHForgeFlow — APPROVED** [review]
Works good. Just a minor remark?

File: account_invoice_inter_company/tests/inter_company_invoice.xml (hunk 1 above)
Comment (AdriaGForgeFlow) [reply account_invoice_inter_company/tests/inter_company_invoice.xml]: Yes, you're right I missed this one.

**Comment by JordiBForgeFlow** [comment]
@AdriaGForgeFlow after reviewing the purchase_sale_inter_company and the previous versions of this module it seems to me that we should better define an inter_company_invoice_user_id, and also we need to add to the config the option to disable the product security rules. Implemented here:
b173f0b

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Functional and technical review.

**Comment by AdriaGForgeFlow** [comment]
> @AdriaGForgeFlow after reviewing the purchase_sale_inter_company and the previous versions of this module it seems to me that we should better define an inter_company_invoice_user_id, and also we need to add to the config the option to disable the product security rules. Implemented here:
> [b173f0b](https://github.com/OCA/multi-company/commit/b173f0b5c890b5569140e57387fb872a17c90a3a)

Makes sense, thanks for the help :+1: 
Is it ready to merge?

**Pull request merged**